src/language-server/config/config.ts

```diff
@@ -29,8 +30,9 @@ function ignoredFieldWarning(
 export interface Context {
   apiKey?: string;
   serviceName?: string;
+  configPath?: string;
 }
-const context = new Slot<Context>();
+const contextStore = new AsyncLocalStorage<Context>();
```

Comment:
Switching to `AsyncLocalStorage` here for consistency - I need to use that in other places already.